apps/prairielearn/elements/pl-image-capture/pl-image-capture.js

```diff
@@ -571,7 +582,7 @@ const MAX_ZOOM_SCALE = 5;
       }
     }
 
-    setHiddenCaptureInputValue(dataUrl) {
+    async setHiddenCaptureInputValue(dataUrl) {
```

Comment:
When we are calling this elsewhere in this file, we aren't using `await`. Is this a problem?